### PR TITLE
add phone number rules for `+17215201993、+5164518135…

### PR DESCRIPTION
**Pull request by leoython**

### CODE REVIEW

File: lib/iso3166Data.js [hunk 1]
```diff
@@ -1880,5 +1880,389 @@ module.exports = [
...
+            8
+        ]
+	},
+	{
+        "alpha2": "US",
+        "alpha3": "USA",
+        "country_code": "1",
+        "country_name": "United States",
+        "mobile_begin_with": [
+            "201",
```
Comment (Bossa573) [inline lib/iso3166Data.js]: please reduce no. of lines similar to other countries

File: lib/iso3166Data.js [hunk 2]
```diff
@@ -1880,5 +1880,389 @@ module.exports = [
 		country_name: 'Zimbabwe',
 		mobile_begin_with: ['71', '73', '77'],
 		phone_number_lengths: [9]
-	}
+	},
+	{
+        "alpha2": "SX",
+        "alpha3": "SXM",
```
Comment (Bossa573) [inline lib/iso3166Data.js]: duplicated country

File: lib/iso3166Data.js [hunk 3]
```diff
@@ -1880,5 +1880,389 @@ module.exports = [
...
+        ],
+        "phone_number_lengths": [
+            10
+        ]
+	},
+	{
+        "alpha2": "PE",
+        "alpha3": "PER",
+        "country_code": "51",
+        "country_name": "Peru",
```
Comment (Bossa573) [inline lib/iso3166Data.js]: duplicated country

File: lib/iso3166Data.js [hunk 4]
```diff
@@ -1880,5 +1880,389 @@ module.exports = [
...
+            "6"
+        ],
+        "phone_number_lengths": [
+            9,
+            8
+        ]
+	},
+	{
+        "alpha2": "MY",
+        "alpha3": "MYS",
```
Comment (Bossa573) [inline lib/iso3166Data.js]: duplicated country

File: lib/iso3166Data.js [hunk 5]
```diff
@@ -1880,5 +1880,389 @@ module.exports = [
...
+        ],
+        "phone_number_lengths": [
+            9,
+            10,
+            8
+        ]
+	},
+	{
+        "alpha2": "US",
+        "alpha3": "USA",
```
Comment (Bossa573) [inline lib/iso3166Data.js]: duplicated country

**Comment by Bossa573** [comment]
1. Please check the existing country and modify the record if needed
2. if you need to add the new country, please ordering it in alphabetical order
3. Please reduce number of lines similar to other entries

File: lib/iso3166Data.js (hunk 1 above)
Comment (leoython) [reply lib/iso3166Data.js]: ok,thanks

File: lib/iso3166Data.js [hunk 6]
```diff
@@ -1336,8 +1336,14 @@ module.exports = [
 		alpha3: 'PER',
 		country_code: '51',
 		country_name: 'Peru',
-		mobile_begin_with: ['9'],
-		phone_number_lengths: [9]
+		mobile_begin_with: [
+            "9",
```
Comment (Bossa573) [inline lib/iso3166Data.js]: single quote string

Comment (Bossa573) [inline lib/iso3166Data.js]: from line 1340 to line 1346, you're mixing spaces and tab

**Comment by Bossa573** [comment]
https://github.com/AfterShip/phone/releases/tag/2.3.20

**Pull request merged**